packages/components/src/DocumentViewer/DocumentViewer.tsx

```diff
@@ -94,40 +102,74 @@ export class DocumentViewer extends React.Component<IProps, IState> {
     }
   }
 
+  zoomIn = () => {
+    this.setState((prevState) => ({ ...prevState, zoom: prevState.zoom + 0.2 }))
+  }
+
+  zoomOut = () => {
+    this.setState((prevState) => {
+      if (prevState.zoom >= 1) {
+        return { ...prevState, zoom: prevState.zoom - 0.2 }
+      } else {
+        return prevState
+      }
+    })
+  }
+
+  rotateLeft = () => {
+    this.setState((prevState) => ({
+      rotation: (prevState.rotation - 90) % 360
+    }))
+  }
+
   render() {
     const { options, children, id } = this.props
+    const isSupportingDocumentsEmpty =
+      this.state.selectedDocument && this.state.selectedDocument.length > 0
 
     return (
-      <Container id={id}>
-        {options.documentOptions.length > 0 && (
-          <>
-            <Select
-              id="select_document"
-              options={options.selectOptions}
-              color="inherit"
-              value={this.state.selectedOption as string}
-              onChange={(val: string) => {
-                const imgArray = options.documentOptions.filter((doc) => {
-                  return doc.label === val
-                })
-                if (imgArray[0]) {
-                  this.setState({
-                    selectedOption: val,
-                    selectedDocument: imgArray[0].value
+      <ViewerWrapper id={id}>
+        <>
+          <ViewerContainer>
+            <ViewerHeader>
+              <Select
+                id="select_document"
+                options={options.selectOptions}
+                color="inherit"
+                value={this.state.selectedOption as string}
+                onChange={(val: string) => {
+                  const imgArray = options.documentOptions.filter((doc) => {
+                    return doc.label === val
                   })
-                }
-              }}
-            />
-            {this.state.selectedDocument && (
-              <DocumentImage
-                id="document_image"
-                image={this.state.selectedDocument}
+                  if (imgArray[0]) {
+                    this.setState({
+                      selectedOption: val,
+                      selectedDocument: imgArray[0].value
+                    })
+                  }
+                }}
+              />
+              <PanControls
+                zoomIn={this.zoomIn}
+                zoomOut={this.zoomOut}
+                rotateLeft={this.rotateLeft}
               />
+            </ViewerHeader>
+            {isSupportingDocumentsEmpty && (
+              <ViewerImage>
+                <PanViewer
+                  key={Math.random()}
```

Comment:
Wonder why we've had Math.random() as the key here.. it doesn't persist between renders. I think we can just remove this line